index.html

```diff
@@ -0,0 +1,246 @@
+        <link rel="stylesheet" href="./style/body/banner/__text/color/banner__text_color_orange.css"/>
+        <link rel="stylesheet" href="./style/body/banner/__filter/position/banner__filter_position_top-left.css"/>
+        <link rel="stylesheet" href="./style/body/banner/__filter/position/banner__filter_position_top-right.css"/>
+        <link rel="stylesheet" href="./style/body/banner/__filter/position/banner__filter_position_bottom-left.css"/>
+        <link rel="stylesheet" href="./style/body/banner/__filter/position/banner__filter_position_bottom-right.css"/>
+        <link rel="stylesheet" href="./style/body/title/title.css"/>
+        <link rel="stylesheet" href="./style/body/title/__text/title__text.css"/>
+        <link rel="stylesheet" href="./style/body/title/button/button.css"/>
+        <link rel="stylesheet" href="./style/body/title/button-with-image/button-with-image.css"/>
+        <link rel="stylesheet" href="./style/body/row/row.css"/>
+        <link rel="stylesheet" href="./style/body/row/content/content.css"/>
+        <link rel="stylesheet" href="./style/body/row/content/__text/content__text.css"/>
+        <link rel="stylesheet" href="./style/body/row/content/__circle-with-img/content__circle-with-img.css"/>
+        <link rel="stylesheet" href="./style/body/subscribe/subscribe.css"/>
+        <link rel="stylesheet" href="./style/body/subscribe/__text/subscribe__text.css"/>
+        <link rel="stylesheet" href="./style/body/subscribe/subscribe-form/subscribe-form.css"/>
+        <link rel="stylesheet" href="./style/body/subscribe/subscribe-form/__input/subscribe-form__input.css"/>
+        <link rel="stylesheet" href="./style/body/models/models.css"/>
+        <link rel="stylesheet" href="./style/body/models/panel/panel.css"/>
+        <link rel="stylesheet" href="./style/body/models/panel/__content/panel__content.css"/>
+        <link rel="stylesheet" href="./style/body/footer/footer.css"/>
+        <link rel="stylesheet" href="./style/body/footer/__content/footer__content.css"/>
+        <link rel="stylesheet" href="./style/body/footer/__content/__text/footer__text.css"/>
+        <link rel="stylesheet" href="./style/body/footer/__content/link-button/link-button.css"/>
+    </head>
+    <body class="body">
+        <header class="header">
+            <a href="" class="logo">
+                <span class="text_size_big logo__name">WEBSITE-NAME</span>
+                <span class="text_size_small logo__slogan">Jevakiro puso od ia javai</span>
+            </a>
+            <ul class="menu">
+                <li><a class="menu__item home-button" href="">HOME</a></li>
+                <li><a class="menu__item" href="#about">ABOUT</a></li>
+                <li><a class="menu__item" href="#services">SERVICES</a></li>
+                <li><a class="menu__item" href="#portfolio">PORTFOLIO</a></li>
+                <li><a class="menu__item" href="#contact">CONTACT</a></li>
```

Comment:
If we add a few additional menu items, they should be displayed on the same line:  https://www.screencast.com/t/SxCa3YIEM